x/gov/types/v1/params.go

```diff
@@ -27,6 +27,9 @@ var (
 	DefaultMinInitialDepositRatio    = sdk.ZeroDec()
 	DefaultProposalCancelRatio       = sdk.MustNewDecFromStr("0.5")
 	DefaultProposalCancelDestAddress = ""
+	DefaultBurnProposal              = false // set to false to replicate behavior of when this change was made (0.47)
```

Comment:
```suggestion
	DefaultBurnProposalPrevote              = false // set to false to replicate behavior of when this change was made (0.47)
```
nit: just to follow consistency

and also is this default value going to be set to `true` in the coming releases? if yes please create an issue for this, ignore if not.

Reply:
might as well leave it and allow chains to set it. They can also modify it in the migration